File: clients/typescript/tests/e2e/e2b-corpus/bound-client.mjs
// Sigue https://docs.e2b.dev/client ("SDK client").
import assert from "node:assert/strict";
import { E2B } from "rayito/e2b";

const client = new E2B();
const sbx = await client.Sandbox.create();
try {
  assert.equal((await sbx.runCode("1 + 1")).text, "2");
} finally {
  await sbx.kill();
}
console.log("bound-client ok");

File: clients/python/tests/e2e/e2b_corpus/sync/bound_client.py
# Sigue https://docs.e2b.dev/client ("SDK client").
from rayito.e2b import E2B

client = E2B()
sbx = client.Sandbox.create()
try:
    assert sbx.run_code("1 + 1").text == "2"
finally:
    sbx.kill()
print("bound_client ok")
